.changeset/long-dots-grab.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux/ui5-application-inquirer': minor
+'@sap-ux/inquirer-common': minor
+---
```

Comment:
ui5-libr
```suggestion
'@sap-ux/project-input-validator': patch
'@sap-ux/ui5-library-inquirer': patch
---
```

Reply:
Hmm, I think it should be minor or major for first release no?

Reply:
For the new modules sure. But I am suggesting to add patches for the other 2 modules changed.

Reply:
Isn't @devinea saying to add the following and keep minor for the others
```
@sap-ux/project-input-validator': patch
@sap-ux/ui5-library-inquirer': patch
```